front/components/agent_builder/AgentBuilderTemplate.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  ArrowPathIcon,
+  Button,
+  Markdown,
+} from "@dust-tt/sparkle";
+import React, { useContext } from "react";
+import { useFormContext } from "react-hook-form";
+
+import type { AgentBuilderFormData } from "@app/components/agent_builder/AgentBuilderFormContext";
+import { ConfirmContext } from "@app/components/Confirm";
+import type { FetchAssistantTemplateResponse } from "@app/pages/api/templates/[tId]";
+
+interface AgentBuilderTemplateProps {
+  assistantTemplate: FetchAssistantTemplateResponse;
+}
+
+export function AgentBuilderTemplate({
+  assistantTemplate,
+}: AgentBuilderTemplateProps) {
+  return (
+    <div className="flex-1 overflow-y-auto p-4">
+      <div className="flex flex-col gap-4">
+        <TemplateButtons assistantTemplate={assistantTemplate} />
+        {assistantTemplate.helpInstructions && (
+          <>
+            <Markdown content={assistantTemplate.helpInstructions} />
+            {assistantTemplate.helpActions && (
+              <div className="h-px bg-border" />
+            )}
+          </>
+        )}
+        {assistantTemplate.helpActions && (
+          <Markdown content={assistantTemplate.helpActions} />
+        )}
+      </div>
+    </div>
+  );
+}
```

Comment:
Can we maybe add some labels to clarify which field corresponds to what? Locally (with very dummy data I agree) I'm a bit lost
<img width="939" height="306" alt="Screenshot 2025-08-13 at 08 33 06" src="https://github.com/user-attachments/assets/d2163511-30d2-4cb9-822a-33139187d03b" />